core/data.go

```diff
@@ -121,9 +121,10 @@ func (b *BlobHeader) EncodedSizeAllQuorums() int64 {
 
 // BlomCommitments contains the blob's commitment, degree proof, and the actual degree.
 type BlobCommitments struct {
-	Commitment  *Commitment `json:"commitment"`
-	LengthProof *Commitment `json:"length_proof"`
-	Length      uint        `json:"length"`
+	Commitment       *Commitment       `json:"commitment"`
```

Comment:
I wonder if it would be clearer/cleaner to just call the `Commitment` and `LengthCommitment` fields `G1Commitment` and `G2Commitment`? 

Wdyt @bxue-l2, @jianoaix, @ian-shim?

Reply:
For the name of the type, yeah that sounds clearer